scipy/stats/_continuous_distns.py

```diff
@@ -3079,6 +3079,90 @@ def _stats(self, mu):
 invgauss = invgauss_gen(a=0.0, name='invgauss')
 
 
+class norminvgauss_gen(rv_continuous):
+    r"""A Normal Inverse Gaussian continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `norminvgauss` is:
+
+    .. math::
+
+        f(x; a, b) = 
+        \frac{a  \exp\left(\sqrt{a^2 - b^2} + b x\right)}
+        {\pi \sqrt{1 + x^2}} \, K_1\left(a * sqrt{1 + x^2}\right)
```

Comment:
We had a discussion about TeX style a while ago in https://github.com/scipy/scipy/pull/7756

In short: let's treat TeX markup as code, and apply the 'readability counts' principle. Here I'd suggest to ditch `\left` and `\right` and replace `\frac{...}{...}` by `(...) / (...)`